tasks/configuration.yml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
Future request: think please about how to dynamically add any filters that you need. At the current state You can't rewrite existent filters as well as add new

Reply:
For now - nothing, therefore delete it.
Add later, if nessesary